flink-core/pom.xml

```diff
@@ -33,6 +33,25 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<properties>
+		<surefire.module.config> <!--
+			required by JmxServer
+			-->--add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED <!--
+			PluginConfigTest
+			-->--add-opens=java.base/java.util=ALL-UNNAMED <!--
+			ExceptionUtilsTest
+			-->--add-opens=java.base/java.lang=ALL-UNNAMED <!--
+			-->--add-opens=java.base/java.lang.invoke=ALL-UNNAMED <!--
+			StateDescriptorTest
+			-->--add-opens=java.base/java.io=ALL-UNNAMED <!--
```

Comment:
because of `import.io.File`? :thinking:

Reply:
because it uses kryo to serialize a `File`, whatever that is supposed to do :shrug: